src/functions.ts

```diff
@@ -37,6 +37,19 @@ export function equals(a, b) {
   return a !== a && b !== b;
 }
 
+export function isInteger(value: number): boolean {
+  return Math.floor(value) == value
+}
+
+export function isIntegerArray(array: number[]): boolean {
+  for (let value of array) { if (!isInteger(value) && value != null) return false };
```

Comment:
Consider using `array.some()` (equivalent to python's `any`)

Reply:
Changed in feat/logscale

Reply:
To avoid merge actions